### PR TITLE
Feature/back campaign

### PR DESCRIPTION
**Pull request by bitTal**
This PR styles the Campaign main page including the admin section footer and the shared header.

### CODE REVIEW

File: app/views/layouts/admin.html.erb [hunk 1]
```diff
@@ -4,14 +4,28 @@
   <title>Onboarding</title>
   <%= csrf_meta_tags %>
 
-  <%= stylesheet_link_tag 'admin', media: 'all', 'data-turbolinks-track': 'reload' %>
-  <%= javascript_include_tag 'admin', 'data-turbolinks-track': 'reload' %>
+  <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (bitTal) [inline app/views/layouts/admin.html.erb]: I copied this code from the other PR, so this part is wrong here, but I realised and switched it again in the next commits.

File: app/views/shared/_header.html.erb [hunk 2]
```diff
@@ -1,20 +1,16 @@
-<nav class="navbar navbar-dark bg-primary">
+<nav class="l-header wrap-full">
   <button class="navbar-toggler hidden-sm-up" type="button" data-toggle="collapse" data-target="#exCollapsingNavbar2">
     &#9776;
   </button>
-  <div class="collapse navbar-toggleable-xs" id="exCollapsingNavbar2">
```
Comment (tiagojsag) [inline app/views/shared/_header.html.erb:5]: Be careful when changing partials, they might be used in more than one place. This in particular is used both on the home and on the admin, and it seems to me that this is now broken on both of them.

File: app/assets/stylesheets/admin.scss [hunk 3]
```diff
@@ -10,14 +10,18 @@
...
-body {
-  padding-top: 20px;
-  padding-bottom: 20px;
-}
+@import 'bootstrap';
 
-.navbar {
-  margin-bottom: 20px;
-}
+// body {
```
Comment (tiagojsag) [inline app/assets/stylesheets/admin.scss]: if it's not needed, remove this please

File: app/views/shared/_header.html.erb (hunk 2 above)
Comment (bitTal) [inline app/views/shared/_header.html.erb:5]: It's already implemented with the mobile design, have a look when you can. @tiagojsag

**Pull request closed without merging**